libre-computer/aml-a311d-cc/dt/i2c-b.dts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2017 BayLibre, SAS.
+ * Author: Neil Armstrong <narmstrong@baylibre.com>
+ *
+ * SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+ */
+
+/*
+ * Overlay aimed to enable I2C_B on Header 7J1 :
+ * Pins 27 (SDA), 28 (SCL)
+ */
+
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "amlogic,aml-a311d-cc";
+
+	fragment@0 {
+		target-path = "/aliases";
+
+		__overlay__ {
+			i2c0 = "/soc/cbus@c1100000/i2c@87c0";
```

Comment:
How would I determine which address to use? AFAICT it should be a free address range which will then be used to map the i2c data into memory.